<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stack_pages">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>801</width>
      <height>541</height>
     </rect>
    </property>
    <widget class="QWidget" name="page_settings">
     <widget class="QGroupBox" name="group_microphone">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>371</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string>Microphone</string>
      </property>
      <widget class="QComboBox" name="dropdown_devices">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>251</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Device</string>
       </property>
      </widget>
      <widget class="QSlider" name="slider_threshold">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>251</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="pageStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>300</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Sensitivity</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="group_preferences">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>361</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string>Preferences</string>
      </property>
      <widget class="QComboBox" name="dropdown_voiceType">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Default</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Voice Type</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Audio Responses</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_main">
     <widget class="QPushButton" name="btn_speak">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>361</width>
        <height>341</height>
       </rect>
      </property>
      <property name="text">
       <string>Speak to NATALIE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>390</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Output</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkbox_listen">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>60</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Listening...</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textbox_output">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>410</y>
        <width>361</width>
        <height>101</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>My response will be here...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="field_input">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>181</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_navigation">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_main">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>0</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>NATALIE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_settings">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>0</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
